go/store/types/codec.go

```diff
@@ -374,7 +386,7 @@ const (
 
 func (b *binaryNomsWriter) ensureCapacity(n uint32) {
 	length := uint64(len(b.buff))
-	minLength := uint64(b.offset + n)
```

Comment:
this was overflowing, and then being cast to uint64.  Need to cast before adding.